ctapipe/io/containers.py

```diff
@@ -75,6 +75,10 @@ class RawCameraContainer(Container):
     """
     Storage of raw data from a single telescope
     """
+    event_number = Item(-1, "telescope event number")
+    num_channels = Item(-1, "number of channels per pixel")
```

Comment:
Thanks for your prompt comments.
Concerning the field appearing in the InstrumentContainer, I though this container will be deprecated, but indeed if it is not I can use it.

Pixel flag is indeed a specific to our camera. I just realise that, that would probably mean I should create camera specific DataContainer class. In our case it gives the status of each pixel in a given event. What do you think of this? When I look in the DataModel of CamerasToACTL there is indeed fields which are different from one camera to the other, which we should be able to readout. Maybe that would just be for an expert version (and correspond more to the R1 data format)

Concerning the event_number and event_id, that's a good question. To me right now it is the event counter from the DigiCam I am testing. I don't think that there is a mechanism that allow this event counter to be the same than the future overall cta event number. Since the Event_id in RawDataContainer counts for all telescope I was thinking it would be usefull to have the telescope "local" counter in the RawCameraContainer. But maybe if we go for camera specific container as discussed before I should put it there.

Let me know what you think about it